packages/calcite-components/src/components/segmented-control/segmented-control.e2e.ts

```diff
@@ -193,6 +193,33 @@ describe("calcite-segmented-control", () => {
     expect(await getSelectedItemValue(page)).toBe("2");
   });
 
+  it("updates selection when cleared with undefined", async () => {
+    async function getSelectedItemValue(page: E2EPage): Promise<string> {
+      return page.$eval(
+        "calcite-segmented-control",
+        (segmentedControl: HTMLCalciteSegmentedControlElement) => segmentedControl.selectedItem.value,
+      );
+    }
+    const page = await newE2EPage();
+    await page.setContent(
+      `<calcite-segmented-control>
+            <calcite-segmented-control-item value="1" checked>one</calcite-segmented-control-item>
+            <calcite-segmented-control-item value="2">two</calcite-segmented-control-item>
+          </calcite-segmented-control>`,
+    );
+    await page.waitForChanges();
```

Comment:
Nitpick: for readability, can you add newlines between the test setup, and each update/assert chunk?